interface/main/messages/save.php

```diff
@@ -128,9 +128,9 @@
 								PHONE_country_code,LABELS_local,LABELS_choice)
 							VALUES (?,?,?,?,?,?,?,?,?,?)";
             sqlStatement($sqlINSERT, array($response['customer_id'], $response['API_key'], $_POST['new_email'], $facilities, $providers, "1", "1", "1", "1", "5160"));
-            sqlQuery("UPDATE `background_services` SET `active`='1',`execute_interval`='5', `require_once`='/library/MedEx/MedEx_background.php' WHERE `name`='MedEx'");
-
-            $info = $MedEx->login('1');
+            $query = "UPDATE `background_services` SET `active`='1',`execute_interval`='?, `require_once`='/library/MedEx/MedEx_background.php' WHERE `name`='MedEx'";
```

Comment:
this query will likely fail since there is a lone apostrophe to the left of the `?` character